### PR TITLE
Added Clipboard Module

### PR DESCRIPTION
**Pull request by Vinochs**

### CODE REVIEW

File: packages/clipboard/.circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version: 2
```
Comment (mnmtanish) [inline packages/clipboard/.circleci/config.yml:1]: Delete `.circleci`, we use Github Actions in this repo.

File: packages/clipboard/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "clipboard",
```
Comment (mnmtanish) [inline packages/clipboard/package.json:2]: Should be `@creately/clipboard`

File: packages/clipboard/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export function add(x: number, y: number = 0) {
```
Comment (mnmtanish) [inline packages/clipboard/src/index.ts:1]: Please add actual code and unit tests

File: package.json [hunk 4]
```diff
@@ -3,5 +3,9 @@
   "private": true,
   "scripts": {
     "publish-all": "node ./.scripts/publish-all"
+  },
+  "dependencies": {
```
Comment (mnmtanish) [inline package.json:7]: Do not make any changes to the root `package.json` file or the `package-lock.json` file. Create a new one inside `packages/clipboard`.

File: packages/clipboard/src/index.ts [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+          return text;
+        })
+        .catch(() => of(this.retriveLocalClipboardData()))
+    );
+  }
+
+  /**
+   * Store data into the local storage
+   */
+  protected storeToLocalClipboard(data: any) {
```
Comment (mnmtanish) [inline packages/clipboard/src/index.ts:50]: Please make these `private`, we're not opening them for others to extend. For unit tests, we can do something weird like `as any` if we want to use these methods.

File: packages/clipboard/src/index.ts [hunk 6]
```diff
@@ -0,0 +1,72 @@
+import { Observable, from, of } from 'rxjs';
+import * as clipboardpoly from 'clipboard-polyfill';
```
Comment (mnmtanish) [inline packages/clipboard/src/index.ts:2]: This is a dependency, should be there on `package.json` file

File: packages/clipboard/src/index.ts [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+ * to other browser window.
+ * Local clipboard provides ability to copy or paste the data within the application.
+ */
+export class Clipboard {
+  /**
+   * This copies the given data to the system clipboard and as fallback,
+   * it copies to local storage.
+   * @param data data that needs to be copied to the clipboard
+   */
+  public copy(data: any): Observable<any> {
```
Comment (mnmtanish) [inline packages/clipboard/src/index.ts:20]: This doesn't have to be an observable. Please use promises. We can get rid of `rxjs` from this module.

File: packages/clipboard/src/index.ts (hunk 6 above)
Comment (Vinochs) [reply packages/clipboard/src/index.ts:1]: it's there in the `package.json` file of clipboard module.

File: packages/clipboard/src/index.ts (hunk 7 above)
Comment (Vinochs) [reply packages/clipboard/src/index.ts]: Done.

File: packages/clipboard/src/index.ts (hunk 6 above)
Comment (mnmtanish) [reply packages/clipboard/src/index.ts:1]: Ok, sorry

**Pull request merged**